ui/src/theme/adminControlsStyles.js

```diff
@@ -4,79 +4,87 @@ import base from 'theme';
 import { adminPillHover } from 'theme/hoverStyles';
 
 const {
-  fonts: { libreFranklin, openSans },
+  fonts: { openSans },
   keyedPalette: {
-    brandPrimary,
+    black,
+    bombay,
     burntSienna,
-    pelorousapprox,
-    valencia,
-    sienna,
-    white,
+    deepIron,
+    elm,
     frenchGrey,
-    lightBlack,
     iron,
-    mystic,
-    black,
-    dustyGray,
+    lightBlack,
     lightPorcelain,
+    mystic,
+    pelorousapprox,
+    sienna,
+    white,
   },
 } = base;
 
-const pillBlue = pelorousapprox;
-const pillRed = valencia;
-const pillBlueDisabled = '#AADBE3';
-const pillRedDisabled = '#E8AFB2';
-const pillOrange = sienna;
-const pillGrey = dustyGray;
-const pillBorderColour = frenchGrey;
+const pillBlue = elm;
+const pillBlueHighlight = pelorousapprox;
+const pillDisabled = deepIron;
+const pillOrange = burntSienna;
+const pillGrey = bombay;
 
 const actionsMenuHighlight = lightBlack;
 const actionsMenuHover = mystic;
 
 const pillColour = ({ secondary, disabled }) => {
-  if (secondary && disabled) {
-    return { base: brandPrimary, hover: brandPrimary };
+  if (disabled) {
+    return { base: white, hover: white };
   } else if (secondary) {
-    return { base: brandPrimary, hover: valencia };
+    return { base: black, hover: black };
   } else {
     return { base: white, hover: white };
   }
 };
 
 const pillBackgroundColour = ({ primary, warning, secondary, disabled, info }) => {
-  if (primary && disabled) {
-    return { base: pillRedDisabled, hover: pillRedDisabled };
+  if (disabled) {
+    return { base: pillDisabled, hover: pillDisabled };
   } else if (primary) {
-    return { base: pillRed, hover: burntSienna };
-  } else if (warning && disabled) {
-    return { base: pillOrange, hover: sienna };
+    return { base: pillBlue, hover: pillBlueHighlight };
   } else if (warning) {
-    return { base: pillOrange, hover: '#D9805A' };
-  } else if (secondary && disabled) {
-    return { base: white, hover: white };
+    return { base: pillOrange, hover: sienna };
   } else if (secondary) {
     return { base: white, hover: lightPorcelain };
   } else if (info) {
-    return { base: pillGrey, hover: '#A2A2A2' };
-  } else if (disabled) {
-    return { base: pillBlueDisabled, hover: pillBlueDisabled };
+    return { base: pillGrey, hover: frenchGrey };
   } else {
-    return { base: pillBlue, hover: '#58BAC9' };
+    return { base: pillBlueHighlight, hover: elm };
+  }
+};
+
+const pillBorderColour = ({ primary, warning, secondary, disabled, info }) => {
```

Comment:
this works, could also write with a switch to stop the if/else ladder annoyance:
```
switch(true){
  case disabled:
    return pillDisabled;
  case primary:
    return pillBlue;
  default:
    return pillBlueHighlight;
}
```

Reply:
(I know you didn't write it this way... I just get irked by long if/else chains lol)

Reply:
Good call, I'll make this change in my colour updates PR